package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "broccoli-less-single",
   "description": "Single-file-output LESS compiler for Broccoli",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Could you remove as well, since usually the author does the bump once ready to release.

Reply:
👍